src/Symfony/Bridge/Twig/Extension/SecurityExtension.php

```diff
@@ -22,10 +23,12 @@
  */
 class SecurityExtension extends \Twig_Extension
 {
+    private $roleChecker;
     private $securityChecker;
 
-    public function __construct(AuthorizationCheckerInterface $securityChecker = null)
+    public function __construct(RoleCheckerInterface $roleChecker, AuthorizationCheckerInterface $securityChecker = null)
```

Comment:
Adding this dependency as first non-optional argument is a BC break in case someone extended the `SecurityExtension` or redefined the service definition.